tests/test_api.py

```diff
@@ -62,15 +63,16 @@ def test_veracode_xml_api_version(self):
     ## VeracodeXMLAPI app_id property
     def test_veracode_xml_api_app_id(self):
         """
-        Test the VeracodeXMLAPI app_id property
+        Test the VeracodeXMLAPI app_name property
         """
-        veracode_xml_api = VeracodeXMLAPI()
+        with patch("veracode.api.get_app_id", return_value=0):
```

Comment:
`app_id` should be a string, not an int.

Reply:
Looks like I originally made `get_app_id` return `Union[int, None]` - I'm assuming that needs updated to `Union[str, None]` then? 

The XML schema says the app_id is a `long` which caused some confusion I think: https://analysiscenter.veracode.com/resource/2.0/applist.xsd

Reply:
Right, https://github.com/SeisoLLC/easy_sast/blob/e220a85c7cdd2594e2f43c11f595a6d3d739175e/tests/constants.py#L49 shows my explicit test to ensure this isn't an int.  This is an example of the variation that I noted at https://github.com/SeisoLLC/easy_sast/blob/e220a85c7cdd2594e2f43c11f595a6d3d739175e/tests/constants.py#L81